core/src/main/java/com/datastrato/gravitino/catalog/EntityCombinedSchema.java

```diff
@@ -24,6 +25,10 @@ public final class EntityCombinedSchema implements Schema {
   // Sets of properties that should be hidden from the user.
   private Set<String> hiddenProperties;
```

Comment:
Perhaps we can handle this not in this PR, we may need a class named 'combinedEntity' to unify the handling of entities from different sources.

Reply:
@mchades what is the meaning here, can you explain more?

Reply:
I mean do we need to provide an abstract class `CombinedEntity<T>` for `EntityCombinedSchema`, `EntityCombinedTable`, and `EntityCombinedTopic`?